sbt-cross/src/main/scala/sbtcross/JVMCross.scala

```diff
@@ -0,0 +1,31 @@
+package sbtcross
+
+import sbt._
+
+import scala.language.implicitConversions
+
+case object JVMPlatform extends Platform {
+  def identifier: String                = "jvm"
+  def sbtSuffix: String                 = "JVM"
+  def enable(project: Project): Project = project.enablePlugins(CrossPlugin)
+  val crossBinary: CrossVersion         = CrossVersion.binary
+  val crossFull: CrossVersion           = CrossVersion.full
+}
+
+trait JVMCross {
+  val JVMPlatform = sbtcross.JVMPlatform
+
+  implicit def JVMCrossProjectBuilderOps(
+      builder: CrossProject.Builder): JVMCrossProjectOps =
+    new JVMCrossProjectOps(builder.crossType(CrossType.Full))
```

Comment:
Rather than hard-coding `Full` as the default here, we should reuse the default that is available in `CrossProject.Builder.crossProjectFromBuilder`. This can easily be done with
```
new JVMCrossProjectOps(builder)
```
letting the implicit conversion from `Builder` to `CrossProject` kick in to satisfy the expected type of the parameter to `new JVMCrossProjectOps`.